### DISCUSSION TITLE
We have reached +70 Stars ⭐

### DISCUSSION THREAD

**Discussion by PiterWeb** [PiterWeb/LibreRemotePlay#42] [Announcements]
## Hi, guys. I'm very happy that Remote Controller has reached +70 Github stars ⭐.

This is all thanks 🙏 to all of you who are supporting not only by giving your star but also to recent contributions like the French translation by @Zorkyx22 .

I know you have already done a lot for me by supporting the project but I would like to get some feedback 💬 from you. For example: possible improvements of the documentation, ...

🚩 I'm also going to take the opportunity to make a pre-announcement that the project is going to be renamed. This is because when I chose the name (not being a native English speaker) I didn't realise that ‘Remote Controller’ is also the TV remote control. 🤣 Also if you have any name suggestion could be nice.

**Comment by Zorkyx22** [comment]
No problem at all! I intended to look into it further tonight after work, but yesterday I had a little trouble figuring out how to connect the host and the client. The code system feels unintuitive.

Maybe documenting this further would be beneficial? I plan on helping out a bit still, so once I get a good grasp on it I can help out!

**Reply by PiterWeb** [reply]
I have added some new [docs](https://github.com/PiterWeb/RemoteController/tree/main/docs) in the docs section. I hope will help you.
